classes/services/PKPSubmissionService.php

```diff
@@ -643,4 +643,129 @@ public function compileToArrayParams($defaultParams, $params = null) {
 		return $result;
 	}
 
+	/**
+	 * Helper function to create a submission object
+	 *
+	 * @param int $contextId
+	 * @param int $submissionId
+	 * @throws Exceptions\InvalidSubmissionException
+	 *
+	 * @return Submission
+	 */
+	protected function getSubmission($contextId, $submissionId) {
```

Comment:
This is starting to impinge on the authorized context concept we're currently using to get submissions. Is this used to get a single submission e.g. based on a URL parameter, or to fetch based on a dependent object, or to fetch several in a single request?